perfkitbenchmarker/providers/kubernetes/provider_info.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Provider info for Kubernetes
+
+"""
+
+from perfkitbenchmarker import provider_info
+
+
+class KubernetesProviderInfo(provider_info.BaseProviderInfo):
+
+  SUPPORTED_BENCHMARKS = ['block_storage_workload', 'cassandra_ycsb',
```

Comment:
For Kubernetes the list of supported benchmarks I provided contains those which we executed and verified that worked properly. This doesn't mean that the rest of them don't work. So in this case there is a differentiation between `supported`, `not_verified` and `not_working` as intended. I think that we could either:
1. change the `SUPPORTED_BENCHMARKS` list to `UNSUPPORTED_BENCHMARKS=['bonnie++', 'mysql']` (bonnie++ is not supported in containers) which won't work for sure
or
2. maintain two lists: `SUPPORTED_BENCHMARKS` and `UNSUPPORTED_BENCHMARKS` and in case user tries to run a benchmark which is not within any of these lists, then some kind of a warning would be displayed.